docs/platforms/apple/common/configuration/graphql-operation-tracking.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: GraphQL Operation Tracking
+sidebar_order: 14
+description: "Enable tracking of GraphQL operation names in HTTP breadcrumbs and failed request events"
+---
+
+When enabled, the SDK extracts the GraphQL operation name from HTTP requests that have `Content-Type: application/json` and contain a JSON body with an `operationName` field. The operation name is then attached to:
+
+- HTTP breadcrumbs as `graphql_operation_name` (when network breadcrumbs are enabled)
+- Failed request events in the context as `graphql.operation_name` (when HTTP client error capture is enabled)
```

Comment:
`m`: We should mention this is enabled by default by the option `enableCaptureFailedRequests` and link to https://docs.sentry.io/platforms/apple/guides/ios/configuration/http-client-errors/ and/or https://docs.sentry.io/platforms/apple/guides/ios/configuration/options/#enableCaptureFailedRequests